checkout/access-checkout-android/src/main/java/com/worldpay/access/checkout/session/request/handlers/SessionRequestHandlerFactory.kt

```diff
@@ -1,5 +1,11 @@
 package com.worldpay.access.checkout.session.request.handlers
 
+import android.content.Intent
+
+/**
+ * A factory class that is initialised with a [SessionRequestHandlerConfig]
```

Comment:
Do not need to say how it is used in our code with this much detail as could be used elsewhere, only explain what this class does.

```
[SessionRequestHandlerFactory] is responsible for creating all the [SessionRequestHandler]
```

Reply:
updated